deploy/services/tanka/README.md

```diff
@@ -1,6 +1,99 @@
-# Tanka
+# Kubernetes deployment via Tanka
 
-See [Tanka](../../../build/deploy/README.md#kubernetes-deployment-via-tanka)
-and the [legacy deployment procedure](../../../build/README.md#deploying-a-dss-instance-via-kubernetes) 
+This folder contains a set of configuration files to be used by
+[tanka](https://tanka.dev/install) to deploy a single DSS instance via
+Kubernetes following the procedures found in the [build](../../../build) folder.
 
+## Architecture
```

Comment:
Architecture and Sizing aren't specific to tanka, so we may want to move the content of those two sections to deploy/architecture.md as long as it's already being moved